lab_4/decode_text_score_eight_test.py

```diff
@@ -62,7 +62,7 @@ def test_decode_text_incorrect_storage(self):
 
         to_decode = generator.generate_text(context, 2)
 
-        bad_inputs = [(), [], 123, None, NGramTrie]
+        bad_inputs = (tuple(), [], {}, 123, None, NGramTrie)
```

Comment:
why explicit tuple() notation? why not just ()?

Reply:
oh, you're right, i'll fix it))